src/curl.c

```diff
@@ -334,22 +345,22 @@ int swupd_curl_get_file_full(const char *url, char *filename,
 
 		local.staging = filename;
 
-		if (lstat(filename, &stat) == 0) {
-			if (resume_ok && resume_download_enabled) {
-				curl_ret = curl_easy_setopt(curl, CURLOPT_RESUME_FROM_LARGE, (curl_off_t)stat.st_size);
-			} else {
-				unlink(filename);
+		if (resume_ok && resume_download_enabled &&
+		    lstat(filename, &stat) == 0) {
```

Comment:
any reason the `lstat(...)` is on a new line?

Reply:
Probably the line was too big and then I change that again. I'll make them in a line

Reply:
Oh, I broke that because I'm used to break in 80cols, but I noticed that we don't do that on swupd. Is there a line length limit I should follow?

Reply:
No, the rule is just to keep it readable. Wrap when it makes it more readable to do so.